osfoffline/filesystem_manager/sync_local_filesystem_and_db.py

```diff
@@ -11,209 +13,185 @@
     DirCreatedEvent,
 )
 from watchdog.observers import Observer
+from sqlalchemy.orm.exc import MultipleResultsFound, NoResultFound
```

Comment:
Various imports in this section are no longer used, most notably deletion events. Is that an intentional design change?